rest-api/requirements.txt

```diff
@@ -24,3 +24,4 @@ protorpc>=0.12.0
 parameterized>=0.6.1
 googlemaps>=2.5.1
 backoff
+coverage
```

Comment:
I think we may want to consider installing this separately in setup_env.sh instead of putting it here... we don't need coverage to be installed when deploying to AppEngine, right?

Reply:
I had originally put it in `setup_env.sh` but CI wouldn't pick up on it - I suppose I could add it in after [this line](https://github.com/all-of-us/raw-data-repository/blob/2ae703fce7064a972ac4456bb389f6383f6750c1/ci/test.sh#L11).  Attaching it to [the circle config file](https://github.com/all-of-us/raw-data-repository/blob/2ae703fce7064a972ac4456bb389f6383f6750c1/circle.yml#L10) doesn't affect the tests b/c they're run inside a virtualenv inside the ci container.

Reply:
Ok, installing it in `ci/test.sh` definitely works; I've removed it from `requirements.txt`